docs/casestudies/ci123.md

```diff
@@ -0,0 +1,162 @@
+
+1. **Templates**: After filling in the Kubernetes native API declaration, users can select a system-created template to declare these scheduling policies. Thanks to Karmada, the system also supports template CRUD.
+2. **Strategies**: Capabilities are visualized for users to select, as shown below.
+3. 
+   ![Capability visualization](../resources/adoptions-ci123-capability-visualization.png "Capability visualization")
+
+### Differentiated Resource Configurations for Multiple Clusters
+
+Workload configurations (for example, container image tags) of multiple clusters are often different. AIML INSTITUTE encapsulates Karmada's override policies for users, as shown in the following figure:
+
+![](../resources/adoptions-ci123-override.png)
+
+### Multi-Cluster Resource State Aggregation
+
+Resources run in multiple clusters. It is critical to aggregate their states to form a unified view. Karmada aggregates states of native Kubernetes resources, which are used to build the platform as follows:
+
+![Capability visualization-1](../resources/adoptions-ci123-unified-view-1.png "Capability visualization-1")
+
+![Capability visualization-2](../resources/adoptions-ci123-unified-view-2.png " Capability visualization-2")
+
+## Integration of Karmada and Existing Systems
+
+It's easy to integrate Karmada with AIML INSTITUTE's existing systems. AriesSurface is the first project that AIML INSTITUTE tries to migrate from a single to multiple clusters. AriesSurface is a pipeline (DAG)-based inspection system used to detect deployment links and cluster data consistency, which are two critical metrics of clusters. With Karmada, AriesSurface is reconstructed into a multi-cluster inspection system that can observe the states of clusters globally. The following figure shows its new architecture:
+
+![](../resources/adoptions-ci123-aries.png)
+
+
+1. The inspection system delivers CRDs to member clusters through Karmada propagation.
+2. The surface controllers of member clusters listen to CRD creation, render a pipeline based on the definition, and calculate DAGs. The controllers execute the DAGs and generate the inspection results.
+3. The control plane runs a detector to collect, aggregate, and calculate the inspection results. Perceptible link states of member clusters allow more effective cross-cluster deployment and scheduling. The following figure shows the Cluster Inspection page in the online system.
+   ![Cluster inspection](../resources/adoptions-ci123-cluster-inspection.png "Cluster inspection")
```

Comment:
Add a blank line between here，so the picture and the word are not crowded in a row.